crates/ledger-types/src/state_accessor.rs

```diff
@@ -31,11 +31,14 @@ pub trait StateAccessor {
     fn l1_view(&self) -> &Self::L1ViewState;
 
     /// Gets a mut ref to the L1 view state.
-    fn l1_view_mut(&mut self) -> &mut Self::L1ViewState;
+    fn get_l1_view_mut(&mut self) -> &mut Self::L1ViewState;
```

Comment:
This is a direct accessor so adding `get_` is unhelpful and implies there's more going on than there is, plus it's inconsistent with `l1_view`.